content/docs/iac/concepts/resources/providers.md

```diff
@@ -91,17 +91,40 @@ Installing package 'random'...
 If you are tracking a package in the project file and installing via `pulumi install`, be sure to remove any generated SDK files from version control and `.gitignore` the SDK directory or the generated files will still be under version control!
 {{% /notes %}}
 
-## Default Provider Configuration
+## Default and Explicit Providers
 
-By default, each provider uses its package’s global configuration settings, which are controlled by your stack’s configuration. You can set information such as your cloud provider credentials with environment variables and configuration files. If you store this data in standard locations, Pulumi knows how to retrieve them.
+Within a Pulumi program, there are two types of providers you can use to declare resources:
 
-For example, suppose you run this CLI command:
+- **Default providers** are not declared in your Pulumi program, and use global configuration settings. Resources created with a default provider do not need the [`provider` option](/docs/iac/concepts/options/provider/) set in the [resource's options parameter](/docs/iac/concepts/options/). This is the most common way to declare Pulumi resources.
+- **Explicit providers** are explicitly declared in your Pulumi program, and use the configuration values you specify when you declare the provider. Resources created with explicit providers must have the `provider` option set in their resource options. The most common use case for explicit providers are multi-environment deployments of cloud infrastructure in a single stack. Explicit providers are themselves Pulumi resources, and their configuration values are [Pulumi inputs](/docs/iac/concepts/inputs-outputs/).
```

Comment:
Not required, but consider adding a simple table that summarizes:

Default Providers vs. Explicit Providers
When to use each
How to configure each
Limitations of each

It would repeat the text but also drum up the most important points.

Reply:
Added